spec/human_player_spec.rb

```diff
@@ -5,20 +5,20 @@
 RSpec.describe HumanPlayer do
   class ValidatorWithNoMethods end
 
-  class UserWithOneValidInput
+  class UserWithFourValidInput
     def valid_input(message:, validator:)
-      1
+      4
     end
   end
 
   describe "#selection" do
     it "returns the player's valid selection" do
       token = ""
       validator = ValidatorWithNoMethods.new
-      user = UserWithOneValidInput.new
+      user = UserWithFourValidInput.new
       player = HumanPlayer.new(token: token, user: user, validator: validator)
 
-      expect(player.selection).to eq(1)
+      expect(player.selection).to eq(4)
```

Comment:
hmm.. curious, why did you need to change this to a 4?

Reply:
I thought I could get one by you.  For no other reason than to give it a unique name.  I should just pull it out into another file like I did FakeDisplay.